packages/ui/react-ui/src/components/Main/Main.tsx

```diff
@@ -41,6 +45,30 @@ type MainContextValue = {
   setComplementarySidebarOpen: Dispatch<SetStateAction<boolean | undefined>>;
 };
 
+const landmarkAttr = 'data-main-landmark';
+
+const useLandmarkMover = (propsOnKeyDown: ComponentPropsWithoutRef<'div'>['onKeyDown'], landmark: string) => {
```

Comment:
interesting name -- can you cite a precedent or just add a comment?

Reply:
'mover' is from tabster: https://tabster.io/docs/mover
'landmark' is a standard term for this kind of element: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/landmark_role
can certainly add a comment to cite these